src/CLI/Task.php

```diff
@@ -40,7 +20,7 @@ class Task
      *
      * @var array
      */
-    protected $labels = [];
+    protected array $labels = [];
```

Comment:
Don't we get this from the Hook class? I think we should.

Reply:
We don't have it in hook, it doesn't make sense in hook does it? What would we use labels in hook for?

Reply:
We can use labels for whatever we want, and it seems like it might be repetitive across servers.

Reply:
labels now in hooks